src/toil/cwl/cwltoil.py

```diff
@@ -1412,6 +1412,13 @@ def remove_unprocessed_secondary_files(unfiltered_secondary_files: dict) -> list
     return final_secondary_files
 
 
+def determine_load_listing(tool):
+    load_listing_req, _ = tool.get_requirement("LoadListingRequirement")
+    load_listing_tool_req = load_listing_req.get("loadListing", "no_listing") if load_listing_req else "no_listing"
+    load_listing = tool.tool.get("loadListing") or load_listing_tool_req
```

Comment:
A tool is never supposed to be able to turn this off if the requirement has it set, right?

Reply:
Fixed!